contentcuration/contentcuration/tests/test_contentnodes.py

```diff
@@ -34,6 +34,11 @@ def _check_nodes(parent, title=None, original_channel_id=None, source_channel_id
 
 
 class NodeOperationsTestCase(BaseTestCase):
+    def setUp(self):
```

Comment:
would it be better to override `setUpClass`?

Reply:
Not in this case, as then any changes to self.channel (e.g. setting the main_tree) would persist across the tests.